lib/python/rose/suite_engine_procs/cylc.py

```diff
@@ -130,19 +130,71 @@ def cmp_suite_conf(
         finally:
             os.unlink(new_suite_rc_processed)
 
-    def gcontrol(self, suite_name, host=None, engine_version=None, args=None):
-        """Launch control GUI for a suite_name running at a host."""
-        if not self.is_suite_registered(suite_name):
-            raise SuiteNotRegisteredError(suite_name)
-        if not host:
-            host = "localhost"
-        environ = dict(os.environ)
-        if engine_version:
-            environ.update({self.get_version_env_name(): engine_version})
-        fmt = r"nohup cylc gui --host=%s %s %s 1>%s 2>&1 &"
+    def gcontrol(self, suite_name, args=None):
```

Comment:
``gcontrol()`` & ``gscan()`` are now largely duplicates. Can we create a separate method e.g. ``launch_gui()`` for consolidation?

Reply:
Fair enough.

Reply:
Updated with a service function.